docs/ubuntu/setup-testnet-validator-from-scratch.md

```diff
@@ -5,7 +5,7 @@
 > the [Test Net Code of Conduct and Incentive Requirements](../testnet.md). Failure to do so may reduce or fully 
 > disqualify any Test Net incentive participation.
 > 
-> Before you set up your node, make sure it conforms to the minimum [Recommended Hardware Specifications](https://docs.casperlabs.io/en/latest/node-operator/hardware.html)
+> Before you set up your node, make sure it conforms to the minimum [Recommended Hardware Specifications](https://casper.network/docs/operators/hardware)
```

Comment:
This is the wrong file to change, it should be done in the `/src/` folder. @mrkara can you help update this? The actual change itself is good. Thanks @JusTTeam .